cli/client.go

```diff
@@ -292,6 +295,52 @@ func (c *AppClient) selectOrganization(orgStr string) error {
 	return nil
 }
 
+// GetOrg gets an org by an indentifying string. If the orgStr is an
+// org UUID, then this matchs on organization ID, otherwise this will match
+// on organization name.
+func (c *AppClient) GetOrg(orgStr string) (*apppb.Organization, error) {
+	if err := c.ensureLoggedIn(); err != nil {
+		return nil, err
+	}
+	resp, err := c.client.ListOrganizations(c.c.Context, &apppb.ListOrganizationsRequest{})
+	if err != nil {
+		return nil, err
+	}
+	organizations := resp.GetOrganizations()
+	var orgIsID bool
+	if _, err := uuid.Parse(orgStr); err == nil {
+		orgIsID = true
+	}
+	for _, org := range organizations {
+		if orgIsID {
+			if org.Id == orgStr {
+				return org, nil
+			}
+		} else if org.Name == orgStr {
+			return org, nil
+		}
+	}
+	return nil, errors.Errorf("no organization found for %q", orgStr)
+}
+
+// GetOwnedOrgByPublicNamespace searches the logged in users orgs to see
+// if any have a matching public namespace.
+func (c *AppClient) GetOwnedOrgByPublicNamespace(publicNamespace string) (*apppb.Organization, error) {
```

Comment:
The name for this is awkward. I originally had `GetOrgByPublicNamespace` but that is not entirely accurate